app/src/main/AndroidManifest.xml

```diff
@@ -18,9 +18,11 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:allowBackup="false"
-        tools:replace="android:allowBackup">
+        tools:ignore="GoogleAppIndexingWarning">
```

Comment:
What does ignoring this do? I couldn't find any docs online.

Reply:
It silences the lint warning. It doesn't do anything other than that.